src/Microsoft.Diagnostics.Monitoring.RestServer/appsettings.json

```
@@ -1,7 +1,20 @@
 {
   "Logging": {
     "LogLevel": {
-      "Default": "Warning"
+      "Default": "Information",
+      "Microsoft": "Warning",
+      "Microsoft.Hosting.Lifetime": "Information"
+    },
+    "Console": {
+      "IncludeScopes": true,
+      "TimestampFormat": "HH:mm:ss "
```

```suggestion
      "FormatterName": "json",
      "FormatterOptions": {
          "SingleLine": true,
          "IncludeScopes": true,
          "TimestampFormat": "HH:mm:ss ",
          "UseUtcTimestamp": true
      }
```

Can we switch to Json logger instead?

---

To be done in a subsequent change as I described in the PR message. I'll need to research which format is best for the analytics platform, but it will most likely be the JSON format. I was also thinking of keeping the default to be the Default formatter and then have the JSON formatter be used in the container that we'll ship. We can discuss more of this offline as needed.

---

👍 Apologies, didn't see your comment. I got too excited to see this PR!